staging/kube-prometheus-stack/.helmignore

```diff
@@ -30,3 +30,5 @@ unittests/
 upgrade_operator.sh
 kube-prometheus-stack-*.tgz
 patch/
+CONTRIBUTING.md
+UPGRADING.md
```

Comment:
I don't see `UPGRADING.md` in this branch or in master - does that matter?

Reply:
https://github.com/mesosphere/charts/blob/master/staging/kube-prometheus-stack/UPGRADING.md

Reply:
ahhhhhhhh
gooootcha. My error.

Reply:
I mean - I knew it had to be somewhere. I just didn't know where.